src/qcodes/instrument_drivers/cryomagnetics/Cryomagnetics4G.py

```diff
@@ -0,0 +1,379 @@
+from __future__ import annotations
```

Comment:
I think we should remame this module `_cryomagnetics4G.py` to signal that the drivers should only be imported from the toplevel public api. I wish that I had done this with all the other drivers

Reply:
good catch, I'll make that change.